### ISSUE TITLE
audiobooks: UI controls, accessibility

### ISSUE THREAD

**Issue by danielweck** [readium/readium-desktop#942]
Currently, the UI controls rendered by the navigator include the title, cover image, buttons for play/pause/resume, previous/next, forward/rewind, and the interactive local time progress indicator / scrub bar timeline.
These are keyboard-accessible, but not labelled with multilingual strings for screen readers.

Follow-up to PR https://github.com/readium/readium-desktop/pull/938

**Comment by danielweck** [comment]
Moved keyboard shortcut issue separately here: https://github.com/edrlab/thorium-reader/issues/1107

**Comment by danielweck** [comment]
Conversely, a TTS readaloud experience (currently inline highlights or simple captioning mode) could be presented as pure audiobook player, leveraging the existing GUI or the one that will eventually replace it.
